content/docs/configuration/acme/README.md

```diff
@@ -346,9 +346,29 @@ In this case the `DNS01` solver for Cloudflare will only be used to solve a
 challenge for a DNS name if the `Certificate` has a label from
 `matchLabels` _and_ the DNS name matches a zone from `dnsZones`.
 
+## Private ACME Servers
+
+cert-manager should also work with private or self-hosted ACME servers, as long as they follow the ACME spec.
+
+If your ACME server doesn't use a publicly trusted certificate, you can pass a trusted CA to use when creating your
+issuer, from cert-manager 1.11 onwards:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    ...
```

Comment:
Is it worth including the `server` here as well as the docs are about a custom sever, so this is a slightly more complete example?

```yaml
spec:
  acme:
    server: https://customer.acme.domain.com
    caBundle: <base64 encoded CA Bundle in PEM format>
```

Reply:
Yeah that does make sense, thanks! I'll do that.